content/millennium/r4/individuals/related-person.md

```diff
@@ -0,0 +1,105 @@
+
+The distinction between Practitioner and RelatedPerson is a practitioner cares for multiple patients on behalf of a healthcare facility where a RelatedPerson performs care tasks for a specific patient and is not associated with any healthcare facility.
+
+* [RelatedPerson id](http://hl7.org/fhir/R4/resource-definitions.html#Resource.id){:target="_blank"}
+* [RelatedPerson Identifier](http://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.identifier){:target="_blank"}
+* [Active](https://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.active){:target="_blank"} 
+* [Patient](http://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.patient){:target="_blank"}
+* [Relationship](http://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.relationship){:target="_blank"}
+* [Related person's name](http://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.name){:target="_blank"}
+* [Telecom information (may include phone and email)](http://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.telecom){:target="_blank"}
+* [Gender (administrative)](http://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.gender){:target="_blank"}
+* [Date of birth](http://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.birthDate){:target="_blank"}
+* [Address](http://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.address){:target="_blank"}
+* [Communication (preferred language)](https://hl7.org/fhir/r4/relatedperson-definitions.html#RelatedPerson.communication){:target="_blank"}
+
+## Terminology Bindings
+
+<%= terminology_table(:related_person, :r4) %>
+
+## Search
+
+Search for RelatedPersons that meet supplied query parameters:
+
+    GET /RelatedPerson?:parameters
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true) %>
+
+### Parameters
+
+ Name         | Required?                                         | Type          | Description
+--------------|---------------------------------------------------|-----------------------------------------------------------------------------------------------------------
+ `_id`        | This or `patient` or `identifier` | [`token`]     | The logical resource id associated with the resource.
+ `identifier` | This or `_id` or `patient`        | [`token`]     | A RelatedPerson identifier.
+ `patient`    | This or `_id` or `identifier`     | [`reference`] | A reference to a patient related which the RelatedPerson is associated. Example: `14067892`
+
+Notes:
+
+- `identifier` value must include both a system and a code. Example: `identifier=urn:oid:2.16.840.1.113883.3.13.6|URN:CERNER:...:PI98N2FK5TN`
+
+### Headers
+
+ <%= headers %>
+
+### Example
+
+#### Request
+
+    GET https://fhir-open.sandboxcerner.com/R4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/RelatedPerson?_id=633923-633923
```

Comment:
doesn't seem like the best example we could give when both id components are the same value

Reply:
added new example data

https://github.com/cerner/fhir.cerner.com/pull/254/commits/a6cc265db68cabf4cb1d9abc1dec64f71d1c0b86#diff-b47a4cc256449f5aa96d433a01e1ae01R64

https://github.com/cerner/fhir.cerner.com/pull/254/commits/a6cc265db68cabf4cb1d9abc1dec64f71d1c0b86#diff-e77269d0d7c7c9767d78a8e4f8e121a0

format json:
https://github.com/cerner/fhir.cerner.com/pull/254/commits/b7a8040f281660e9f5f68edf2059c17e67e8509d#diff-e77269d0d7c7c9767d78a8e4f8e121a0R818

Reply:
shrink down json 
https://github.com/cerner/fhir.cerner.com/pull/254/commits/a61676ce8e25adc46975ea912b5016edde8a8b6e#diff-e77269d0d7c7c9767d78a8e4f8e121a0L41